bin/install_test_requirements.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Install requirements and ckanext
+pip install -r /__w/ckanext-stadtzh-harvest/ckanext-stadtzh-harvest/requirements.txt
+pip install -r /__w/ckanext-stadtzh-harvest/ckanext-stadtzh-harvest/dev-requirements.txt
+pip install -e /__w/ckanext-stadtzh-harvest/ckanext-stadtzh-harvest/
+pip install -U requests[security]
+
+# Install ckanext dependencies
+pip install -e git+https://github.com/ckan/ckanext-dcat.git#egg=ckanext-dcat && \
+  pip install -e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest && \
```

Comment:
only if you have to touch your PR again: can you move the pips 🎶  to the left 🎶 ?

Reply:
Ah, I thought I'd indent them to show that this is all one inline command... but I think the two-space indent just makes it look messy. :sweat_smile: Maybe I'll put them all on separate lines now that they're in this script, not the `test.yml`.

Reply:
Fixed. :)